CesiumGS/cesium
# Use gpu for e2e tests

# Description

I was having some issues running our end to end tests where it spiked my CPU to 100% and took 20-60+ seconds _per test_ which would mean they take over an hour to complete. After this change it brings most tests down to ~2-5 seconds and only a very small number taking longer than 10 seconds which makes the whole suite take closer to 15 minutes now.

The changes I made were based on this article along with a few others I found: https://www.createit.com/blog/headless-chrome-testing-webgl-using-playwright/







## Issue number and link

No issue





## Testing plan

- Run `npm run build` to make sure it's built
- Run `npm run test-e2e-update` and `npm run test-e2e`
    - make sure they complete in a reasonable timeframe
    - Check the screenshots in `Specs/e2e/webgl-check` and make sure they show what you'd expect. For example the webgl report screenshots should not have a "Major performance caveat" as mine did before adding the chrome argument.



# Author checklist
